config/default/chainqueryconfig.toml

```diff
@@ -1,60 +1,84 @@
+#SUGGESTED: "lbry:lbry@unix(/var/run/mysqld/mysqld.sock)/chainquery"
+#apimysqldsn=
 
 #API Host and Port is required for the API Server to bind and listen on.
-#DEFAULT-apihostport="0.0.0.0:6300"
+#DEFAULT: "0.0.0.0:6300"
+#apihostport=
 
 #Profile mode enables and disables the reporting of a profile for chainquery
-#DEFAULT-profilemode=false
+#DEFAULT: false
+#profilemode=
 
 #Daemon mode tells chainquery how hard it should work catch up processing the blockchain
-#DEFAULT-deamonmode=0 #BeastMode it continuously process block after block until caughtup.
+#deamonmode=0 #BeastMode it continuously process block after block until caughtup.
 #daemonmode=1 #SlowAndSteadyMode it will process block with a frequency of 1 block every 100ms
 #daemonmode=2 #DelayMode it will process a block with a configured delay frequency (set via 'processingdelay')
 #daemonmode=3 #DaemonMode it will process a block every iteration of the daemon.
+#DEFAULT: 0
+#deamonmode=
 
 #Default client timeout is for communication with the api of chainquery
-#DEFAULT-defaultclienttimeout=20 #Measured in seconds
+#DEFAULT: 20 #Measured in seconds
+#defaultclienttimeout=
 
 #Processing delay is used to determine how frequently chainquery should process a block
 # It is only used if Daemon mode is set to delay mode
-#DEFAULT-processingdelay=100 #Measured in milliseconds
+#DEFAULT: 100 #Measured in milliseconds
+#processingdelay=
 
 #Daemon delay is the frequency at which chainquery checks for work to do.
-#DEFAULT-daemondelay=1 #Measured in seconds
+#DEFAULT: 1 #Measured in seconds
+#daemondelay=
 
 #Profiling options - will output the time take for certain opertions related to the below category
-#DEFAULT-daemonprofile=false
-#DEFAULT-lbrycrdprofile=false
-#DEFAULT-mysqlprofile=false
+#DEFAULT: false (for all 3 params)
+#daemonprofile=
+#lbrycrdprofile=
+#mysqlprofile=
 
 #Slack Hook URL allows slack integration. All logging info level and above is posted to a slack channel.
-#DEFAULT-slackhookurl=[unset]
+#DEFAULT: ""
+#slackhookurl=
 
 #Slack Channel is the channel that you want the messages to appear. Works together with the hook url.
-#DEFAULT-slackchannel=[unset]
+#DEFAULT: ""
+#slackchannel=
 
 #Slack Log Level tells chainquery what level of logging will be sent to the slack channel. It will log all levels below
 # it as well. Panic=0,Fatal=1,Error=2,Warning=3,Info=4,Debug=5
-#DEFAULT-slackloglevel=0
+#DEFAULT: 0
+#slackloglevel=
 
 #The command that should be executed to trigger a self update of the software. For linux, for example, `<yourscript>.sh`
-#DEFAULT-autoupdatecommand=[unset]
+#DEFAULT: ""
+#autoupdatecommand=
 
 #Twilio service of chainquery to send specifically important information to key users of the Chainquery install.
-#DEFAULT-twiliosid=""
-#DEFAULT-twilioauthtoken=""
-#DEFAULT-smsrecipients=["",""]
-#DEFAULT-smsfromphonenumber=""
\ No newline at end of file
+#DEFAULT:
+##twiliosid=""
+##twilioauthtoken=""
+##smsrecipients=["",""]
+##smsfromphonenumber=""
```

Comment:
i'm skeptical of the 4 lines above. perhaps each var should have its own description and default value to remain consistent

Reply:
I think it's fine. They are a group of related parameters. We can add more of a description if needed, but I don't think we should separate them. Maybe we can group them into a toml object instead? Not sure that would be a valuable effort.